sb-cb-ui-consumption/library/sunbird-cb/consumption/src/lib/_common/competency-passbook-mdo/competency-passbook-mdo.component.html

```diff
@@ -31,8 +31,8 @@
                     <div class="line mt-3 mt-2"></div>
                     <div class="flex flex-wrap gap-3 pt-4 chip-container">
                         <ng-container *ngFor="let child of allcompetencyTheme[obj.name]?.children| slice:0: allcompetencyTheme[obj.name]?.viewMore ? allcompetencyTheme[obj.name]?.children?.length : 3; let i = index">
-                            <div class="chip rounded-full p-2 text-xs chip-ellipsis" [title]="child?.name" >
-                                {{ child.name }}
+                            <div class="chip rounded-full p-2 text-xs chip-ellipsis" [title]="child?.displayName" >
+                                {{ child.displayName }}
```

Comment:
please put the optional check , same as above

Reply:
Added the check